zappa/cli.py

```diff
@@ -2278,164 +2292,168 @@ def create_package(self, output=None):
 
         with zipfile.ZipFile(handler_zip, 'a') as lambda_zip:
 
-            settings_s = "# Generated by Zappa\n"
+            settings_s = self.get_zappa_settings_string()
```

Comment:
All of [this code](https://github.com/Miserlou/Zappa/blob/818f7de2aee18fb2f85e074207e034240dc9e242/zappa/cli.py#L2281-L2430) has been copied directly to `self.get_zappa_settings_string()` so the same settings string can be sent to an environment variable instead of written to `zappa_settings.py` (as it is done in zip-package deployments)